OpenSim/Common/TimeSeriesTable.h

```diff
@@ -434,10 +444,11 @@ class TimeSeriesTable_ : public DataTable_<double, ETY> {
         return row;
     }
     /**
-     * Trim TimeSeriesTable to rows that have times that lies between 
-     * newStartTime, newFinalTime. The trimming is done in place, no copy is made. 
+     * Trim TimeSeriesTable to rows that have times that lies between newStartTime,
+     * newFinalTime inclusive. The trimming is done in place, no copy is made.
```

Comment:
"inclusively"?

Reply:
Also, consider rephrasing to "...lies between newStartTime and newFinalTime, inclusively".

Reply:
I looked it up and I think "(inclusive)" is the best grammar and it was somewhere else it the file already